### PR TITLE
Adding some more researchers

### PR DESCRIPTION
**Pull request by jcnorby**

### CODE REVIEW

File: _people/AimyWissa.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Aimy
+last_name: Wissa
+title: Assistant Professor of Mechanical Science and Engineering
+school: University of Illinois at Urbana-Champaign
+website: https://mechanical.illinois.edu/directory/faculty/awissa
+image: https://robotics.illinois.edu/files/2019/10/Wissa-Aimy.jpg
+---
+Prof. Wissa directs the [Bio-inspired Adaptive Morphology Laboratory](https://bamlab.mechse.illinois.edu/) has pioneered the field of bio-inspired robotics, particularly in the areas of avian-inspired wing design, insect-like locomotion, and multi-functional morphing structures.
```
Comment (aaronjoh) [inline _people/AimyWissa.md:9]: AND has pioneered...
though its not like bio-inspired is new, so maybe "pioneered many new ideas in the field of bio-inspired..."

File: _people/AndrewWilliams.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Andrew
+last_name: Williams
+title: Associate Dean for Diversity, Equity and Inclusion
+school: University of Kansas
+website: https://engr.ku.edu/andrew-williams#link1
+image: https://engr.ku.edu/sites/engr.ku.edu/files/williams_andrew.jpg
+---
+Prof. Williams directs the Humanoid Engineering & Intelligent Robotics (HEIR) Lab. His research interests are in humanoid robotics and AI, intelligent humanoid coaches, and cooperative autonomous systems. He founded the SpelBots and the Advancing Robotics Technology for Societal Impact Alliance and authored the book “Out of the Box: Building Robots, Transforming Lives.”
```
Comment (aaronjoh) [inline _people/AndrewWilliams.md:9]: The last sentence should probably have links, especially to the book. (Actually, why isnt the book one of the featured citations?)

File: _people/ConradTucker.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Conrad
+last_name: Tucker
+title: Arthur Hamerschlag Career Development Professor of Mechanical Engineering
+school: Carnegie Mellon University
+website: https://engineering.cmu.edu/directory/bios/tucker-conrad.html
+image: https://www.meche.engineering.cmu.edu/_files/images/research-groups/tucker-group/conradt.png
+---
+Prof. Tucker leads [Artificial Intelligence in Products Engineered for X (AiPEX) Lab](https://www.meche.engineering.cmu.edu/faculty/aipex.html). the explores the use of machine learning methods that predictively improve the outcome of product design solutions through the acquisition, fusion and mining of large-scale, publicly-available data.
```
Comment (aaronjoh) [inline _people/ConradTucker.md:9]: Sentence structure after lab link

Comment (aaronjoh) [reply _people/ConradTucker.md:9]: Also comma after "fusion"

**Comment by jcnorby** [comment]
See changes.

**Pull request merged**